sdk/cosmos/azure-cosmos/samples/config.py

```diff
@@ -5,4 +5,7 @@
     'master_key': os.environ.get('ACCOUNT_KEY', '[YOUR KEY]'),
     'database_id': os.environ.get('COSMOS_DATABASE', '[YOUR DATABASE]'),
     'container_id': os.environ.get('COSMOS_CONTAINER', '[YOUR CONTAINER]'),
+    'tenant_id': os.environ.get('TENANT_ID', '[YOUR CONTAINER]'),
+    'client_id': os.environ.get('COSMOS_CONTAINER', '[YOUR CONTAINER]'),
+    'client_secret': os.environ.get('COSMOS_CONTAINER', '[YOUR CONTAINER]'),
```

Comment:
I'm thinking the client_id and client_secret aren't the same thing as the "your container"?